package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@segment/analytics.js-core",
   "author": "Segment <friends@segment.com>",
-  "version": "3.13.4",
+  "version": "3.13.5",
```

Comment:
I have pushed this version manually, from my local env, to npm, but package.json has not been pushed before